recipes/whylogs-sketching/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "whylogs-sketching" %}
+{% set version = "3.4.1.dev2" %}
```

Comment:
This needs to target the dev channel: https://conda-forge.org/docs/maintainer/knowledge_base.html#pre-release-builds

Reply:
Haven't done this yet. Will do soon.